VariablesHomework/VariablesHomework.cs

```diff
@@ -20,38 +27,48 @@ public void Declaration()
         public void Assignment()
         {
             int i = 0;
-            Assert.IsTrue(i == 25); // Make this pass
+            Assert.IsTrue(i == 0); // Make this pass
```

Comment:
I expected to change the value of i rather than to change the Assert (and similar for some of the other Asserts below), but this works.

Reply:
ah! ok, gotcha. I wasn't totally sure what my limitations were for making the changes. I am going back through right now, to changing them coorespondingly. thanks!